ios/fastlane/README.md

```
@@ -87,6 +87,22 @@ Run tests on Beta
 
 Run tests on Production
 
+### ios get_version_alpha
```

markdownlint-cli2 (0.17.2)

Heading style
Expected: setext; Actual: atx

(MD003, heading-style)